### PR TITLE
Add storybook and first stories

### PR DESCRIPTION
**Pull request by cedricmessiant**
Storybook config, doc and first stories

### CODE REVIEW

File: assembl/static2/.storybook/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```
Comment (swergas) [inline assembl/static2/.storybook/webpack.config.js]: Why is this file needed in folder .storybook?

Comment (swergas) [reply assembl/static2/.storybook/webpack.config.js]: Oh is it because storybook runs its own web server?

Comment (vincentfretin) [reply assembl/static2/.storybook/webpack.config.js]: yes, it starts its own webpack server on port 9001

**Comment by cedricmessiant** [comment]
@swergas we need to extend default storybook webpack config to have scss files support See https://storybook.js.org/configurations/custom-webpack-config/#extend-mode
